internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/fastclient/utils/AbstractClientEndToEndSetup.java

```diff
@@ -255,7 +252,9 @@ protected void prepareData() throws Exception {
     storeVersionName = creationResponse.getKafkaTopic();
     storeName = Version.parseStoreFromKafkaTopicName(storeVersionName);
     veniceCluster.useControllerClient(
-        client -> client.updateStore(storeName, new UpdateStoreQueryParams().setStorageNodeReadQuotaEnabled(true)));
+        client -> client.updateStore(
+            storeName,
+            new UpdateStoreQueryParams().setStorageNodeReadQuotaEnabled(true).setReadComputationEnabled(true)));
```

Comment:
we should add a few integration tests for compute?

Reply:
Yeah, good call. I also found some bugs due to this. Thanks